src/components/EditPhoto.jsx

```diff
@@ -39,35 +40,45 @@ export const EditPhoto = ({ handleUpdate, showInput, shouldShowInput }) => {
       handleUpdate(imageUrl);
       setImageUrl("");
     } catch (error) {
-      <span>Heelo my friend</span>;
       console.log(error);
+      setError("invalid url");
     }
   };
 
   return (
-    <>
+    <div>
       <ClickAwayListener onClickAway={hideInputClickAway}>
-        <TextField
-          id="outlined-basic"
-          label="Paste your new URL image here"
-          variant="outlined"
-          value={imageUrl}
-          onChange={(e) => setImageUrl(e.target.value)}
-        />
+        <div>
+          <TextField
+            id="outlined-basic"
+            label="Paste your new URL image here"
+            variant="outlined"
+            value={imageUrl}
+            onChange={(e) => setImageUrl(e.target.value)}
+          />
+          <Button
+            variant="contained"
+            color="success"
+            style={{ margin: "10px" }}
+            onClick={verifyIfImageUrlIsValid}
+          >
+            Send edit
+          </Button>
+          <Snackbar
+            open={!!error}
```

Comment:
if you want to use the `open` variable, just change this form `open={!!error}` to `open={open}`

Reply:
I personally would use something like a `Option<string>`